recipe/meta.yaml

```diff
@@ -81,7 +79,7 @@ outputs:
         - networkx >=3.0
         - dill
         - pywin32  # [win]
-        - kokkos =4.7.* h*  # [linux]
+        - kokkos =4.7.01 h*  # [linux]
         - kokkos =4.4.01 h*  # [not linux]
```

Comment:
do we really need to stick to the 4.4.01 on non linux?
the 4.7.01 is available for all platforms.

https://anaconda.org/channels/conda-forge/packages/kokkos/overview

Reply:
for the rest, looks perfect. Thanks @Laureleen06 for the work ;)